### PR TITLE
now using async

### PR DESCRIPTION
**Pull request by flokol120**
Co-Authored-By: Bruno Scheufler <brunoscheufler@users.noreply.github.com>

### CODE REVIEW

File: src/misc/helper.ts [hunk 1]
```diff
@@ -89,22 +118,22 @@ export const saveBinaryHex = (
...
-const createDir = () => {
-  if (!existsSync(JITCOIN_DIR)) {
-    mkdirSync(JITCOIN_DIR);
+const createDir = async () => {
+  if (!(await pathExists(JITCOIN_DIR))) {
+    await mkdirp(JITCOIN_DIR);
   }
-  if (!existsSync(BLOCKCHAIN_DIR)) {
-    mkdirSync(BLOCKCHAIN_DIR);
+  if (!await pathExists(BLOCKCHAIN_DIR)) {
```
Comment (timweiss) [inline src/misc/helper.ts]: Is there a reason why this call is not wrapped in parentheses? Let's keep it consistent.
```suggestion
  if (!(await pathExists(BLOCKCHAIN_DIR))) {
```

**Comment by timweiss** [comment]
Just another suggestion: I think async arrow functions add a lot of noise, which would be reduced with using normal functions. Take a look at and let's decide on how to proceed development https://stackoverflow.com/questions/34361379/arrow-function-vs-function-declaration-expressions-are-they-equivalent-exch

**Comment by flokol120** [comment]
> Just another suggestion: I think async arrow functions add a lot of noise, which would be reduced with using normal functions. Take a look at the link and let's decide on how to proceed development https://stackoverflow.com/questions/34361379/arrow-function-vs-function-declaration-expressions-are-they-equivalent-exch

I personally prefer arrow functions over normal ones but this is something to be discussed in the weekly scrum I assume. :smile:.

**Comment by timweiss** [comment]
Okay, I'll create a Jira ticket and we'll discuss it next time.

**Pull request merged**